invisible_cities/io/mcinfo_io.py

```diff
@@ -168,6 +172,37 @@ def __call__(self, mctables: (tb.Table, tb.Table, tb.Table, tb.Table),
         self.generator_table.flush()
 
 
+def copy_mc_info(files_in, h5out, event_list):
+    """
```

Comment:
I think this function can go to components, it seems not general enough for io module to me. Also add type hints